actors.md

```diff
@@ -1636,3 +1632,20 @@ func getTransaction(txid UInt) Transaction {
 	return tx
 }
 ```
+
+```go
+func AggregateBitfields(faults []FaultSet) Bitfield {
+  var out Bitfield
+  for _, f := range faults {
+    out = out.Union(f.bitField)
+  }
+  return out
+}
+```
+
+```go
+func BurnFunds(amt TokenAmount) {
+  TransferFunds(BurntFundsAddress, amt)
```

Comment:
Where do we define `BurntFundsAddress`?

Reply:
It's defined as address 99 somewhere, I think at the top of this file.